### ISSUE TITLE
STOP POSTING ABOUT AMONG US!

### ISSUE THREAD

**Issue by HardWare68** [HardWare68/CollabDiscordBot#6]
I'M TIRED OF SEEING IT! MY FRIENDS ON TIKTOK SEND ME MEMES, ON DISCORD IT'S FUCKING MEMES! I was in a server, right? and ALL OF THE CHANNELS were just among us stuff. I-I showed my champion underwear to my girlfriend and t-the logo I flipped it and I said 'hey babe, when the underwear is sus HAHA DING DING DING DING DING DING DING DI DI DING' I fucking looked at a trashcan and said 'THAT'S A BIT SUSSY' I looked at my penis I think of an astronauts helmet and I go 'PENIS? MORE LIKE PENSUS

**Comment by HardWare68** [comment]
@BlaisinX im talking about you

**Comment by BlaisinX** [comment]
why don't you get among some cleaning products And take a fucking shower

**Comment by HardWare68** [comment]
Stop posting about taking a shower! I'm tired of seeing it! My friends on TikTok take showers, on Discord it's showers! I was in a server, right? and ALL OF THE CHANNELS were just shower stuff. I-I showed my girlfriend my bathroom and t-the shower I turned it on and said "Hey babe, when the shower is sus! HAHA! DING DING DING DING DING DING DING DING DING!" I looked at a trashcan and said "that's me lol" I looked at my penis I think of the shower and I go "Penis? More like PENSUS!"
